### ISSUE TITLE
Persisted identifier link errror for some of our vocab terms

### ISSUE THREAD

**Issue by UbuntuScott** [geological-survey-of-queensland/vocabularies#428]
Hi @KellyVance 

We've found an error in our catalogue that relates back to an issue in Vocprez 

When clicking on the persisted identifier link for one of the terms in the qg-security-classifications vocab, we get an error. 
For example: 
https://vocabs.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/qg-security-classifications/official
^ this page is ok
But when I click on the persistent identifier link 
An error page appears with the following message: 

Input Error

Status
    400
Message
    The object with URI https://linked.data.gov.au/def/qg-security-classifications/official is not of type skos:ConceptScheme, skos:Collection or skos:Concept and only these classes of object are understood by VocPrez

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Can you please help fix? 
There could be an IRI update or we may have to update the TTL file for the qg-security classsifcations vocab. 

Please advise

Thanks 
@UbuntuScott  
cc @nicholascar  @broughd-qld 
(I recall you two chatting about something related.

**Comment by nicholascar** [comment]
These are all related to the HTTP/HTTPS issues. We had to upgrade linked.data.gov.au to HTTPS (Fed Gov rules!) but not all linked.data.gov.au IRI data in files has been upgraded.

**Comment by UbuntuScott** [comment]
Thanks @nicholascar  I figured this one might have been sitting in the HTTP/HTTPS camp. 
Is there anything required from our side?
fixes to ttl files etc? 
Wondering if I can help?

**Comment by nicholascar** [comment]
The complete solution is just to find/replace all `http://linked.data.gov.au` with `https://linked.data.gov.au` in all the RDF files in this repo. It may be that there is a downstream system at GSQ issue here though with the HTTP variants hard-coded into things like the georesources catalogue. So this solution needs staging and testing at GSQ. But the initial find/replace fix should take about 2 min in a reasonable search-in-all-files text editor!

**Comment by UbuntuScott** [comment]
Easy! we'll take stock and review our vocabs next week and see if we can push some updates through 
Thanks Nick!

**Comment by UbuntuScott** [comment]
reminder to self - update the modified dates too. @UbuntuScott

**Comment by UbuntuScott** [comment]
Seems there's a whole bunch of URI/ IRI's  that point back to TERN 
e.g. 
https://linkeddata.tern.org.au/viewer/tern/id/http://linked.data.gov.au/def/tern-cv/d62a4664-89a1-49a1-9ef0-f30ee2c6d22b
We have a lot of these in our Observed Properties vocab.   @nicholascar  is there someone at TERN updating these?  or an issue / PR I can refer to please? 

I'll try and collate a list of the vocabs which are impacting our Catalogue. 
Thanks

**Comment by nicholascar** [comment]
Yes, @edmondchuc is your man at TERN and he's maintaining things in a similar manner to these things here. He knows about the HTTP/HTTPS upgrade requirements so I guess best just to check with him TERN's expected resolution timeline here.

**Comment by UbuntuScott** [comment]
PR #430 for 3 DES vocabs, that don't appear to require any other teams. 
Will reach out to edmond for the TERN vocabs, which is a dependency to our observed-properties https update

**Comment by nicholascar** [comment]
Am I reading it right that PR #430 just updates the modified date on the 3 vocabs and nothing else?

**Comment by UbuntuScott** [comment]
> Am I reading it right that PR #430 just updates the modified date on the 3 vocabs and nothing else?

I made a few http://linked.data.gov => https:// changes to the branch earlier this morning...  but perhaps the dates didn't get updated.  
The vocprez site was still showing an error. 
This term here is a good example - 
https://vocabs.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/qg-security-classifications/official-public

perhaps there's more to do here than updating our DES vocabs?

**Comment by UbuntuScott** [comment]
I swear there were "http"  's in there this morning when I started this.  But I can't see my commit anywhere!

**Comment by UbuntuScott** [comment]
@KellyVance I think the rest of this fix might need staging and testing with your team, as Nick has advised. 
I'll keep looking through our impacted vocabs. 
Thanks

**Comment by KellyVance** [comment]
@UbuntuScott as per #430 I can update those that are exclusively used by DES.
The fix for the bulk of the vocabularies used by GSQ will need to be managed and staged as mentioned. A lot of dependencies in a number of existing systems that need to be identified and the updates applied simultaneously to keep everything synchronised and functional.

**Comment by UbuntuScott** [comment]
hi @KellyVance  and @nicholascar  
qg-data-licenses.ttl  or https://vocabs.gsq.digital/object?uri=http%3A//linked.data.gov.au/def/licence-document/cc-by-4.0 
is the next one that we've found needs updating.  
In looking at all the components - 
I'm aware that the revision file in https://catalogue.linked.data.gov.au/resource/176 still contains http and not https

[ttl file](https://github.com/geological-survey-of-queensland/vocabularies/blob/master/vocabularies/qg-data-licenses.ttl) in vocprez is ok. 
Front end vocprez and SPARQL has errors

Wondering how I can best help  progress this?
CC @broughd-qld

**Comment by broughd-qld** [comment]
@KellyVance and @nicholascar 

I have created PR #436 to update the next couple of DES vocabularies. The files in GitHub appear OK, but the URIs are failing and VocPrez appears to have HTTP. Hoping a reload corrects the issue.
